tests/ChannelIdempotentTest.php

```diff
@@ -139,6 +142,9 @@ public function testIdempotentMixedIds() {
      * RSL1k4
      */
     public function testIdempotentLibraryGeneratedPublish() {
+        if (self::$testApp->getOptions()['environment'] == 'sandbox') {
+            $this->markTestSkipped('idempotent not available in sandbox environment');
```

Comment:
Idempotent is available in sandbox though. @paddybyers this is correct right?

Reply:
yes, I've reverted that change and idempotent is working in the sandbox